datapackage_pipelines_measure/pipeline_steps/social_media.py

```diff
@@ -1,23 +1,74 @@
 import os
 
-ROOT_PATH = os.path.join(os.path.dirname(__file__), '..', '..')
+from datapackage_pipelines_measure.config import settings
+
+
+DOWNLOADS_PATH = os.path.join(os.path.dirname(__file__), '../../downloads')
 
 label = 'social-media'
 
 
 def add_steps(steps: list, pipeline_id: str,
               project_id: str, config: dict) -> list:
-    return steps + [
-        ('add_resource', {
-            'name': 'test_resource',
-            'url': 'https://docs.google.com/spreadsheets/d/' +
-            '1vbhTuMDNCmxdo2rPkkya9v6X1f9eyqvSGsY5YcxlcLk/' +
-            'edit#gid=0'
-        }),
-        ('stream_remote_resources', {}),
-        ('measure.capitalise', {}),
-        ('dump.to_path', {
-            'out-path':
-                '{}/downloads/{}'.format(ROOT_PATH, pipeline_id)
-        })
-    ]
+    for entity in config['twitter']['entities']:
+        steps.append(('measure.add_twitter_resource', {
+            'entity': entity,
+            'project_id': project_id
+        }))
+
+    steps.append(('concatenate', {
+        'target': {
+            'name': 'social-media',
+            'path': 'data/social-media.json'},
```

Comment:
The path is useful if I want to `dump.to_path` in development. I have a patch lined up that will dump the latest to path, only there's a DEVELOPMENT flag in the env.

The `.json` extension is a mistake, and interestingly, is ignored when the resource file is dumped, and a `.csv` extension is used instead.